docs/source/tutorials/token_classification/tutorial-p2.ipynb

```diff
@@ -0,0 +1,522 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "069a3ce3",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "# %%capture \n",
+    "# !pip uninstall cleanlab -y \n",
+    "# !pip install git+https://github.com/ericwang1997/cleanlab.git@token_classification \n",
+    "# !pip install termcolor "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "33d67688",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# package versions: \n",
+    "# cleanlab==2.0.0 (ericwang/cleanlab -b token_classification)\n",
+    "# termcolor==1.1.0 "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d0d2e007",
+   "metadata": {},
+   "source": [
+    "# Token Classification Label Error Detection - Part 2 \n",
```

Comment:
Let's have title be:

Finding Label Errors in Token Classification Datasets

Reply:
and change filename to be: `token_classification.ipynb`

(make sure to use `git mv` whenever you rename files to keep the git tracking)